Makefile

```diff
@@ -221,3 +209,14 @@ $(DEBIANDIR)/% : debian/%
 	mkdir -p $(DEBIANDIR)
 	cp -f $< $@
 
+#### FORTRAN BINDINGS ####
+
+export NCCL_MAJOR NCCL_MINOR NCCL_PATCH CUDA_MAJOR CUDA_MINOR LIBLINK
+
+forlib : lib
+	$(MAKE) -C fortran lib BUILDDIR=../$(BUILDDIR)
+fortest :
```

Comment:
I don't really need it since inside the fortran Makefile, tests depend on the library.